tensorflow/lite/experimental/delegates/coreml/BUILD

```diff
@@ -56,6 +56,7 @@ objc_library(
         "//tensorflow/lite:kernel_api",
         "//tensorflow/lite:minimal_logging",
         "//tensorflow/lite/c:common",
+        "//tensorflow/lite/delegates:utils",
```

Comment:
This was the missing dependency that made it seem that the CoreML delegate depended on the Metal delegate

Reply:
Oops I missed this while exporting to github. Thanks for catching!